src/doc/trpl/structs.md

```diff
@@ -47,3 +54,36 @@ fn main() {
 ```
 
 This will print `The point is at (5, 0)`.
+
+Rust does not support mutability at the field level, so you cannot write
+something like this:
+
+```rust,ignore
+struct Point {
+    mut x: i32,
+    y: i32,
+}
+```
+
+Mutability is a property of the binding, not of the structure itself. If you’re
+used to field-level mutability, this may seem strange at first, but it
+significantly simplifies things. It even lets you make things mutable for a short
+time only:
+
+
+```rust,ignore
+struct Point {
+    x: i32,
+    y: i32,
+}
+
+fn main() {
+    let mut point = Point { x: 0, y: 0 };
+
+    point.x = 5;
+
+    let point = point; // this new binding is immutable
```

Comment:
We may wan to refrain from the world "immutable" as in theory it could be mutable if it contained a cell. Perhaps something like:

```rust
let point = point; // prevents further mutation of `point`
```

Although it is indeed strictly true that *this* `point` variable is indeed immutable, so it may not be so bad?